src/ray/gcs/gcs_client/service_based_accessor.h

```diff
@@ -9,6 +10,32 @@ namespace gcs {
 
 class ServiceBasedGcsClient;
 
+/// \class Sequencer
```

Comment:
Consider making this class thread safe if it will be called from core worker.

Reply:
fixed